src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -238,6 +238,15 @@
     type: command
     short-summary: delete an application's password or certificate credentials
 """
+helps['ad user create'] = """
+    type: command
+    short-summary: Create an Azure Active Directory user.
+    parameters:
+        - name: --force-change-password-next-login
+          short-summary: Marks this user as needing to update their password the next time they authenticate.
+        - name: --password
+          short-summary: The password that should be assigned to the user for authentication.
+"""
```

Comment:
The linter rule in the past was supressing this rule exception through: https://github.com/Azure/azure-cli/blob/dev/tools/automation/cli_linter/rules/ci_exclusions.yml#L23
Please look at: https://github.com/Azure/azure-cli/blob/dev/tools/automation/cli_linter/util.py#L22

I suspect that moving the modules has bypassed the filtering mechanism used by the linter to exclude modules for certain rule exceptions.

Reply:
Let's see if a516f2d handles this :)

Reply:
Might also want to re-add one of the failing scenarios to test and make sure the change works :)

Reply:
Good idea, I'll add one back. 

Also looks like the change is insufficient because consuming code assumes that each of the command_modules is its own package. I'm curious about how deep that rabbit hole goes, I'll spend a little more time digging.

Reply:
Poking around, it looks like our linter exceptions are really tied to the packages, not modules. I opened up #8656 to track this work, but I can do it after this PR is merged.